notifier/src/main/scala/uk/ac/wellcome/platform/archive/notifier/services/CallbackUrlService.scala

```diff
@@ -13,26 +13,33 @@ import uk.ac.wellcome.platform.archive.display.ResponseDisplayIngest
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
-class CallbackUrlService(contextURL: URL)(implicit actorSystem: ActorSystem,
+class CallbackUrlService(contextUrl: URL)(implicit actorSystem: ActorSystem,
                                           ec: ExecutionContext)
     extends Logging {
   def getHttpResponse(
     callbackNotification: CallbackNotification): Future[Try[HttpResponse]] = {
```

Comment:
Is the Try type still correct?
Maybe just rusty Scala of how `for`s `yield` .

Reply:
Yep – the `for` is yielding a `Future`, which we then wrap in `Success/Failure` as appropriate.

Reply:
Could we not convert the `Try` to a `Future` and then `flatMap` it out to remove the nested structure?